packages/liveblocks-react/src/umbrella-store.ts

```diff
@@ -736,12 +794,23 @@ export class UmbrellaStore<M extends BaseMetadata> {
       (hv) => hv
     );
 
+    const channelNotificationSettings = DerivedSignal.from(
+      this.baseChannelNotificationSettings,
+      this.optimisticUpdates,
+      (baseChannelsNOtificationSettings, updates) =>
```

Comment:
NOticed this tiny typo

```suggestion
      (baseChannelsNotificationSettings, updates) =>
```